### PR TITLE
Fixes the display name issues for tables, and tags count

### PR DESCRIPTION
**Pull request by verdan**
### Summary of Changes

WIP: Made some changes in atlasclient, waiting for them to be fixed and released before this change. 
This PR involves three changes:
- Fixes the table display name for Atlas Proxy. 
- Add support for tags count. 
- Fixes the setup.py requirements package synching issue. 

### Tests

Based on the changes above, I needed to modify the atlas proxy test cases. 

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes `make test`
- [x] I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by verdan** [comment]
@feng-tao @jinhyukchang ready for review

**Comment by feng-tao** [comment]
@verdan  CI fails

**Comment by verdan** [comment]
@feng-tao yes, because the new atlasclient is not yet released. Will be fixed once that's there. Will ping you once done.

**Comment by verdan** [comment]
@feng-tao ready for review. We need to fork the atlasclient as the maintainer was not active enough, and we want to speed up the development on Amundsen-Atlas now.

File: metadata_service/proxy/atlas_proxy.py [hunk 1]
```diff
@@ -18,23 +19,18 @@
 LOGGER = logging.getLogger(__name__)
 
 
+# noinspection PyMethodMayBeStatic
```
Comment (feng-tao) [inline metadata_service/proxy/atlas_proxy.py]: just curious: what is this line about?

File: requirements.txt [hunk 2]
```diff
@@ -59,4 +59,5 @@ neo4j-driver==1.6.0
 neotime==1.0.0
 pytz==2018.4
 statsd==3.2.1
-atlasclient==0.1.7
+pyatlasclient==1.0.0
```
Comment (feng-tao) [inline requirements.txt]: so this will be the new deps？

File: setup.py [hunk 3]
```diff
@@ -1,7 +1,12 @@
+import os
+
 from setuptools import setup, find_packages
 
-__version__ = '1.0.16'
+__version__ = '1.0.17'
 
+requirements_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'requirements.txt')
+with open(requirements_path) as requirements_file:
+    requirements = requirements_file.readlines()
```
Comment (feng-tao) [inline setup.py]: actually there is some concern over pin the deps in https://github.com/lyft/amundsen/issues/80

File: metadata_service/proxy/atlas_proxy.py (hunk 1 above)
Comment (verdan) [reply metadata_service/proxy/atlas_proxy.py]: This makes pycharm not to give warning about the method being static.

File: requirements.txt (hunk 2 above)
Comment (verdan) [reply requirements.txt]: Yes. We'll be working on this one. (for all microservices)

**Comment by feng-tao** [comment]
thanks @verdan !

**Pull request merged**